pkgs/games/humblebundle/antichamber.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, stdenv_32bit, lib, pkgsi686Linux, fetchHumbleBundle, unzip}:
+
+stdenv_32bit.mkDerivation rec {
+  name = "antichamber-1.1";
+
+  src = fetchHumbleBundle {
+    name = "antichamber_1.01_linux_1392664980.sh";
+    machineName = "antichamber_linux";
+    md5 = "37bca01c411d813c8729259b7db2dba0";
+  };
+
+  dontStrip = true;
+  phases = ["installPhase"];
+
+  installPhase = let
+    rpath = with pkgsi686Linux; lib.makeLibraryPath [
+      "$dest/Binaries/Linux"
+      xorg.libX11
+      xorg.libXi
+      stdenv.cc.cc.lib
+      libpulseaudio
+      libvorbis
+      libogg
+      mesa
+    ];
+   in ''
+    set -x
+    dest="$out/share/antichamber"
+
+    # Unpack binaries and data into $dest
+    mkdir -p "$dest"
+    ${unzip}/bin/unzip $src "data/*" -d $dest && [ $? -le 2 ]
```

Comment:
It's an habit I started with archlinux & AUR. I want to unzip in $dest because it is in the nix store partition, while $CWD will be on a temporary partition. For big binary/data archives, this is faster because it remains on the same partition, and mv operations are mostly immediate.

This could however go in the unpackPhase, but the generic build script requires the unpack phase to unpack into the temporary directory used for the build.
It is truly an install phase, unpack is only needed for sources before compilation. There is no such thing here.